src/docs/development/data-and-backend/firebase.md

```diff
@@ -10,7 +10,7 @@ remote configuration, and hosting for your static files.
 
 {{site.alert.note}}
   Some Flutterfire plugins are in development and have not yet
-  reached 1.0. For more info, see the [Flutterfire plugins][] page.
+  reached 1.0. For more info, see the [https://github.com/FirebaseExtended/flutterfire][FlutterFire] page.
```

Comment:
Hi, @robertclarkson! Thanks for catching the broken link. However, this is not how we'd fix it.

Please restore this line:
reached 1.0. For more info, see the [Flutterfire plugins][] page.

Instead, down below, update this line:
[Flutterfire plugins]: {{site.github}}/flutter/plugins/blob/master/FlutterFire.md

to this:
[Flutterfire plugins]: {{site.github}}/FirebaseExtended/flutterfire

Reply:
Sorry yeah, my markdown foo is not so strong! Also i was pressed on time and couldn't see any code hints